src/pages/journal-vouchers/index.js

```diff
@@ -122,17 +124,36 @@ const JournalVoucher = () => {
     toast.success('Record Deleted Successfully')
```

Comment:
make sure that all the messages are platformLabels

Reply:
will not refactor things in diff screens for now